attributes/server_web2.rb

```diff
@@ -1,8 +1,14 @@
 
 default['icinga2']['web2']['enable'] = false
-default['icinga2']['web2']['source_url'] = 'git://git.icinga.org/icingaweb2.git'
-default['icinga2']['web2']['version'] = 'v2.0.0-rc1'
-default['icinga2']['web2']['web_root'] = '/usr/share/icingaweb2'
+default['icinga2']['web2']['install_method'] = 'git'
+default['icinga2']['web2']['package_action'] = 'upgrade' # use 'install' to avoid upgrades
```

Comment:
Wouldn't setting new package version `node['icinga2']['web2']['version']` be sufficient to install the required package version?